printer_mods/PurchenZuPoden/V0SKIRT_W_DISPLAY/README.md

```diff
@@ -0,0 +1,24 @@
+This is my Skirt Mod to add a FYSETC-Mini-12864-Panel Display, Version 2.1 to the V0
+ 
+![alt text](https://github.com/PurchenZuPoden/VoronUsers/blob/master/printer_mods/PurchenZuPoden/V0SKIRT_W_DISPLAY/03.JPG?raw=true)
```

Comment:
Please make these images into relative links, like this:

    ![front view](03.JPG)

Reply:
Fixed!